patternfly/patternfly-elements
# feat(badge): PF1:1

## <Pull request description>




### Related issues

- #2175


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Badge demo page](https://deploy-preview-2186--patternfly-elements.netlify.app/components/badge/demo/) 


### What has changed and why



- Created BaseBadge.ts and BaseBadge.scss
- Made pfe-badge an extension of BaseBadge
- Aligned PFE badge styles with PatternFly v4 badge


### Testing instructions



- View deploy preview
- Check that `<pfe-badge>` matches [PF's badge](https://patternflyelements.org/components/badge/)


### Ready-for-merge Checklist
